articles/migrations/guides/calling-api-with-idtokens.md

```diff
@@ -0,0 +1,254 @@
+- Using [cross-origin authentication](/cross-origin-authentication) (endpoint: `/co/authenticate`). This endpoint is used to authenticate users when the requests come from different domains. 
+
+### Use the Authorization endpoint
+
+In this section we will use an example to showcase the differences in how you get a token with the [Authorization endpoint](/api/authentication#authorize-client). Keep in mind though that no matter which endpoint you want to migrate, the changes are the same, the only thing that differs is the [scopes](#changes-in-scopes) you will specify in the request.
+
+In the example below, we want to use the [GET User by ID endpoint](/api/management/v2#!/Users/get_users_by_id) to retrieve the full profile information of the logged-in user. To do so, first we will authenticate our user (using the [Implicit grant](/api/authentication?http#implicit-grant)) and retrieve the token(s).
+
+On the `Legacy (ID Token)` script you can see an implementation of the old approach that gets an ID Token (and then uses it to call the endpoint). On the `Current (Access Token)` script you can see the new approach that gets an Access Token as well.
+
+<div class="code-picker">
+  <div class="languages-bar">
+    <ul>
+      <li class="active"><a href="#with-id-token" data-toggle="tab">Legacy (ID Token)</a></li>
+      <li><a href="#with-access-token" data-toggle="tab">Current (Access Token)</a></li>
+    </ul>
+  </div>
+  <div class="tab-content">
+    <div id="with-id-token" class="tab-pane active">
+      <pre class="text hljs">
+        <code>
+https://${account.namespace}/authorize?
+  scope=openid
+  &response_type=id_token
+  &client_id=${account.clientId}
+  &redirect_uri=${account.callback}
+  &nonce=CRYPTOGRAPHIC_NONCE
+  &state=OPAQUE_VALUE
+        </code>
+      </pre>
+    </div>
+    <div id="with-access-token" class="tab-pane">
+      <pre class="text hljs">
+        <code>
+https://${account.namespace}/authorize?
+  audience=https://${account.namespace}/api/v2/
+  &scope=openid%20read:current_user
+  &response_type=id_token%20token
+  &client_id=${account.clientId}
+  &redirect_uri=${account.callback}
+  &nonce=CRYPTOGRAPHIC_NONCE
+  &state=OPAQUE_VALUE
+        </code>
+      </pre>
+    </div>
+  </div>
+</div>
+
+The differences that we introduced in order to get an Access Token that can access the Management API are the following:
+- We set the `audience` to `https://${account.namespace}/api/v2/`
+- We asked for the additional scope `read:current_user`
+- We set the `response_type` to `id_token token` so Auth0 will sent us both an ID Token and an Access Token
+
+Once you have the Access Token you can use it to call the endpoint. This part remains the same, nothing else changes in the request except for the value you use as `Bearer` token. The response remains also the same.
```

Comment:
Would be nice to show a sample access_token, highlighting the audience value, the `sub` and the `scopes`.